src/main/drivers/pwm_output.c

```diff
@@ -311,13 +311,14 @@ void motorDevInit(const motorDevConfig_t *motorConfig, uint16_t idlePulse, uint8
 #endif
 
         /* standard PWM outputs */
-        const unsigned pwmRateHz = useUnsyncedPwm ? motorConfig->motorPwmRate : ceilf(1 / (sMin + sLen * 2));
+        // margin of safety is 4 periods when unsynced
+        const unsigned pwmRateHz = useUnsyncedPwm ? motorConfig->motorPwmRate : ceilf(1 / ((sMin + sLen) * 4));
```

Comment:
I agree the magic number of 4 is entirely arbitrary- and I'll move it to a define at the top of the file. It is only used now for deciding prescaler. If a 16bit timer is close to needing the next higher prescaler this will force it up...